release/ray_release/test_automation/state_machine.py

```diff
@@ -16,6 +16,16 @@
 AWS_SECRET_BUILDKITE = "ray_ci_buildkite_token"
 DEFAULT_ISSUE_OWNER = "can-anyscale"
 WEEKLY_RELEASE_BLOCKER_TAG = "weekly-release-blocker"
+NO_TEAM = "none"
+TEAM = [
+    "core",
+    "data",
+    "kuberay",
+    "ml",
+    "rllib",
+    "serve",
+    "serverless",
+]
```

Comment:
why does this team list need to be repeated in two different files? maybe get a `py_library` and import it?

Reply:
oh the other one is duplicated